---
nav_msgs/OccupancyGrid map
geometry_msgs/PoseStamped[] frontiers